README.md

```diff
@@ -96,6 +96,9 @@ SBOM Output Configuration:
                         exists, it will be overwritten.
   -pb, --purl-bom-ref   Use a component's purl for the bom-ref value, instead
                         of a random UUID
+  --licence_output_format {expression,license} 
```

Comment:
indention seams off.
did you actually call the CLI with `-h` and copy-paste the new help page paragraph here?